AdblockPlusSafari/AdblockPlusContainerController.m

```diff
@@ -40,22 +41,9 @@ - (void)viewDidLoad
 
   // This is the simpliest way to print Browser word using custom bold font.
   CGFloat fontSize = self.adblockBrowserLabel.font.pointSize;
-  UIFont *lightFont = [Appearence defaultLightFontOfSize:fontSize];
```

Comment:
@dedecej did you notice these comments? They haven't been addressed. I _really_ want to stick to solving one issue per PR without unrelated changes.